### DISCUSSION TITLE
how to align the central panel horizontally with the left and right panels ?

### DISCUSSION THREAD

**Discussion by Chaojimengnan** [emilk/egui#4112] [Q&A] [answered]
![1](https://github.com/emilk/egui/assets/51994883/f9201941-4b65-4cec-8043-4b46042d6333)

**Comment by technomage** [comment]
I have seen that kind of thing when adding .horizontal or .vertical around content in one panel and not the other.  As long as the 3 panels have the same set of layouts you should see alignment.  Make sure that the alignment of the panels is top to bottom to keep it from centering the content or bottom justifying it.

**Comment by Chaojimengnan** [comment, marked as answer]
Since the center panel has a different `inner_margin` than the side panel, it is not aligned. Simply adjust the `inner_margin` of the center panel to solve the problem
